recipes/kim_edn/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "kim_edn" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bab88a0a890598577a5dabe0ff19ed573ef7a0be547e5a47b15deeb4ec711db5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
```

Comment:
With `noarch`, please use `python >=3.6`.

Reply:
@npavlovikj  I did noarch at first, but it complained that I can not use selector with noarch

Reply:
Right, you can not have `noarch` and `skip` together. Please remove `skip`, and follow the other suggestions.